commit 66cb2c094d187241b8fbdb7f937829e9a6a315b9
Author: Alexey Kardashevskiy <aik@ozlabs.ru>
Date:   Fri Mar 17 00:48:26 2017 +0000

    powerpc/iommu: Pass mm_struct to init/cleanup helpers
    
    [ Upstream commit 88f54a3581eb9deaa3bd1aade40aef266d782385 ]
    
    We are going to get rid of @current references in mmu_context_boos3s64.c
    and cache mm_struct in the VFIO container. Since mm_context_t does not
    have reference counting, we will be using mm_struct which does have
    the reference counter.
    
    This changes mm_iommu_init/mm_iommu_cleanup to receive mm_struct rather
    than mm_context_t (which is embedded into mm).
    
    This should not cause any behavioral change.
    
    Signed-off-by: Alexey Kardashevskiy <aik@ozlabs.ru>
    Reviewed-by: David Gibson <david@gibson.dropbear.id.au>
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1434_66cb2c094d187241b8fbdb7f937829e9a6a315b9.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1434_66cb2c094d187241b8fbdb7f937829e9a6a315b9.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
